### ISSUE TITLE
Handy Support ?

### ISSUE THREAD

**Issue by Kyr4l** [clynamic/joi.how#17]
I see this is compatible with only vibrating toys, based on how the "game" works ("up" and "down" strokes based on the circle size); Would that be possible to make it compatible with the Handy (and potentially Lovense Solace ?)

**Comment by clragon** [comment]
Hi,

Unfortunately I am unsure what would be required to make the app compatible. 
Do these devices work with intiface? Are special instructions required to work with them?

**Comment by Kyr4l** [comment]
yes these devices are supported by intiface, however they are slightly different  compared to vibrating toys, instead of taking a single value they take one for the stroke speed and another for the stroke length.
with joi.how, this could be implemented by using the «large circle» act as the upper stroke while the «small circle» does the downstroke, i'm guessing the lenght could be tuned manually (i don't have any other idea on how to implement this)

**Comment by clragon** [comment]
great, that helps already. I dont have a device to test this, but I will look into it and get back to you. 
if you would like to join our [discord server](https://discord.clynamic.net/), perhaps we can arrange a quick test later?

**Comment by hejekar845** [comment]
Any update on this?
